tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py36, flake8
+envlist = py27, py38, flake8
```

Comment:
The orb image doesn't seem to have py36 available, and this seems a worthwhile change anyways.